Sources/SwiftSyntaxBuilder/Buildables.swift.gyb

```diff
@@ -258,36 +268,38 @@ extension ${node.syntax_kind}: ${expressible_as_type} {
 
 %   end
 % end
-public protocol ExpressibleAsTokenSyntax {
-  func createTokenSyntax() -> TokenSyntax
+% expressible_as_protocols = (SYNTAX_COLLECTION_EXPRESSIBLE_AS_CONFORMANCES.get('TokenSyntax') or [])[:]
+% expressible_as_protocols += (SYNTAX_BUILDABLE_EXPRESSIBLE_AS_CONFORMANCES.get('TokenSyntax') or [])[:]
+% expressible_as_protocols = map(lambda x : 'ExpressibleAs' + x, expressible_as_protocols)
+% if expressible_as_protocols:
+extension TokenSyntax: ${', '.join(expressible_as_protocols)} {
 }
+% end
```

Comment:
@ahoppen added this section after your approval. 

I also discovered, in another PR, that if we don't add `[:]` ad the end, we are copy by reference. 
I'm not sure why it workers the other places. 

Correct me if I'm wrong

Reply:
Oh, that is surprising. Very good catch.

To make sure we don’t hit this again, I think it would be good if we wrapped `SYNTAX_COLLECTION_EXPRESSIBLE_AS_CONFORMANCES.get` in a function. 
E.g.
```python
def get_expressible_as_conformances(conforming_type):
  expressible_as_protocols = (SYNTAX_COLLECTION_EXPRESSIBLE_AS_CONFORMANCES.get(conforming_type) or [])[:]
  expressible_as_protocols += (SYNTAX_BUILDABLE_EXPRESSIBLE_AS_CONFORMANCES.get(conforming_type) or [])[:]
  return map(lambda x : 'ExpressibleAs' + x, expressible_as_protocols)
```

and also use that in the other places.

Reply:
Something like this?